emperor/support_files/js/view.js

```diff
@@ -337,6 +372,125 @@ DecompositionView.prototype._fastInit = function() {
   this.markers.push(cloud);
 };
 
+/**
+ * Parallel plots closely mirroring the shader enabled _fastInit calls
+ */
+DecompositionView.prototype._fastInitParallelPlot = function()
+{
+  var positions, colors, scales, opacities, visibilities, geometry, cloud;
+
+  // We're really just drawing a bunch of line strips...
+  // highly doubt shaders are necessary for this...
+  var vertexShader = `
+    attribute float scale;
```

Comment:
Can you remove this line?

Reply:
Uhh... maybe done?  The line right after scale?

Reply:
Ohhh!  The scale lines!  Done.